bradybridges/jetcake-react-app
# Styling

#### What's this PR do?
Adds updated styling to most of application.  Adds functionality to update profile picture. Brings in changed user information immediately instead of having to refresh the page to see changes. Adds basic footer component. Fixes bugs revolving around editing profile information.
#### How should this be manually tested?
Make sure that you can edit profile information for created accounts. You should not need to refresh the page to see changes in data relating to profile. Look for styling inconsistencies and provide feedback for changes.
#### Any background context you want to provide?
Need to touch up styles a little bit more and add media queries. Add hero component next.
#### What are the relevant tickets?
#7 #26 #23 #27 #22 #25 
#### Screenshots (if appropriate)
![main](https://user-images.githubusercontent.com/25031031/73586380-77c6e600-4469-11ea-91c0-0f16360105e8.PNG)
![profile](https://user-images.githubusercontent.com/25031031/73586381-7990a980-4469-11ea-9d5e-d59c5e0a5c9e.PNG)
